src/plays/keeper/CreateArea.jsx

```diff
@@ -6,6 +6,8 @@ function CreateArea(props) {
     title: "",
     content: ""
   });
+  const title_len = note.title.trim().length;
+  const content_len = note.content.trim().length;
```

Comment:
its good but we can shorten it one more way like
`const disableButton =  note.title.trim().length === 0 && note.content.trim().length === 0`
or in short
`const disableButton =  !note.title.trim() && !note.content.trim()`